### ISSUE TITLE
Minor version not included in namespace

### ISSUE THREAD

**Issue by megies** [FDSN/StationXML#24]
I noticed that the minor version is not included in the namespace, i.e. 1.0 and 1.1 share the same namespace (`http://www.fdsn.org/xml/station/1` -- with full version included as an attribute `version="1.1"`) and thus it's not possible to add (removed) 1.0 tags as foreign tags in 1.1 documents (e.g. `StorageFormat`). Any thoughts on this? I'm used to namespaces including the full version number somehow (e.g. `http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.10`), but no expert on normal practice in this domain by any means.

**Comment by kaestli** [comment]
I agree this is not the typical choice in a situation where neither documents of version 1.0 fully validate against schema 1.1, nor vice versa. (a good reading on the topic is e.g. http://www.xfront.com/Versioning.pdf ) 
However, I guess this should be carefully reconsidered for the migration to version 1.2 *in the light of the changes planned for that revision*.
